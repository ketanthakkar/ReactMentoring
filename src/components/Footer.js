import React from 'react';

const Footer = () => (
    <footer className="footer-section">
      <span className="netflix-text">netflixroulette</span>
    </footer>
)

export default Footer;