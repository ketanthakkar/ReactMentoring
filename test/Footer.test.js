import React from 'react';
import { mount } from 'enzyme';
import Footer from "../src/components/Footer";

describe('should render Footer correctly', () => {
  const tree = mount(<Footer />);
  test('Snapshot test', () => {
    expect(tree).toMatchSnapshot();
  });
});
