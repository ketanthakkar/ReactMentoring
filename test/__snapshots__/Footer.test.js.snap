// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly Snapshot test 1`] = `
<Footer>
  <footer
    className="footer-section"
  >
    <span
      className="netflix-text"
    >
      netflixroulette
    </span>
  </footer>
</Footer>
`;
